rclcpp/include/rclcpp/service.hpp

```diff
@@ -39,19 +43,15 @@ class ServiceBase
 
   RCLCPP_PUBLIC
   ServiceBase(
-    std::shared_ptr<rmw_node_t> node_handle,
-    rmw_service_t * service_handle,
+    std::shared_ptr<rcl_node_t> node_handle,
     const std::string service_name);
 
   RCLCPP_PUBLIC
-  virtual ~ServiceBase();
```

Comment:
Without this the child class's destructor will not be called when deleting a pointer of this type (which I think could happen in the future if it doesn't already): http://stackoverflow.com/a/461224/671658